lib/configfiles/centos8.xml

```diff
@@ -97,7 +98,7 @@ Alias "/.well-known/acme-challenge" "{{settings.system.letsencryptchallengepath}
 			<service type="dns" title="{{lng.admin.configfiles.dns}}">
 				<!--Bind9 -->
 				<daemon name="bind" title="Bind9 nameserver" default="true">
-					<install><![CDATA[yum install bind]]></install>
+					<install><![CDATA[dnf -y install bind]]></install>
```

Comment:
as said, please leave the `-y` out of the command as we have it for apt-based distro's too

Reply:
ah ok, will undo